modules/jaeger-upgrading-es.adoc

```diff
@@ -0,0 +1,12 @@
+////
+This PROCEDURE module included in the following assemblies:
+- jaeger_install/rhbjaeger-updating
+- service_mesh_install/updating-ossm
+////
+
+[id="upgrading_es5_es6_{context}"]
+= Upgrading from Elasticsearch 5 to Elasticsearch 6
+
+There is an issue with certificates when upgrading from Elasticsearch 5 to Elasticsearch 6.
+
+As a workaround, you must delete your Jaeger instance, then recreate the Jaeger instance.  Re-creating the Jaeger instance triggers creating a new set of certificates.
```

Comment:
@JStickler @objectiser @pavolloffay how about adding detailed steps to perform deletion and recreation of the jaeger instance?

We can recreate the jaeger instance in two ways.
* Delete and recreate from the user's CR
```
$ oc delete -f <user-jaeger-cr-file>
$ oc create -f <user-jaeger-cr-file>
```

* Copy generated CR, delete and recreate with generated CR (this can be used where jaeger installed along with ServiceMesh)
```
$ oc get jaeger -n istio-system
NAME     AGE
jaeger   3d21h
$ oc get jaeger jaeger -oyaml -n istio-system > /tmp/jaeger-cr.yaml
$ oc delete jaeger jaeger -n istio-system
$ oc create -f /tmp/jaeger-cr.yaml -n istio-system
```